src/util/fees.cpp

```diff
@@ -26,16 +29,32 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
 
-    if (mode == fee_modes.end()) return false;
+std::string FeeModes(const std::string& delimiter) {
+    std::ostringstream r;
+    bool first = true;
+    for (const auto& i : FeeModeMap()) {
+        r << (first ? "" : delimiter) + i.first;
+        first = false;
+    }
+    return r.str();
+}
 
-    fee_estimate_mode = mode->second;
-    return true;
+bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
+    auto vector = FeeModeMap();
```

Comment:
457396a4091bb3c27b1e982fb7e08442120b15df

Why?

Reply:
You mean why not just do `: FeeModeMap()` directly? Good point.